### ISSUE TITLE
Test accuracy on validation loss or validation test

### ISSUE THREAD

**Issue by hoangdzung** [rusty1s/pytorch_geometric#979]
## ❓ Questions & Help

Currently, I'm evaluating the Diffpool model. 
There are two evaluation strategies which are implemented in `examples\proteins_diff_pool.py` and `benchmark\kernel\main.py`. The first one bases on best validation test, while the other bases on best validation loss. That leads to the difference between two test results. 
Which one I should use? Thank you.

**Comment by rusty1s** [comment]
You are free to use any of those. I am not sure which of them performs better.

**Comment by hoangdzung** [comment]
Do you remember which one you used in your paper?

**Comment by rusty1s** [comment]
The one from the `benchmark/` directory.

**Comment by hoangdzung** [comment]
Thanks for your reply.
